src/python/pants/base/fingerprint_strategy.py

```diff
@@ -78,6 +79,9 @@ class TaskIdentityFingerprintStrategy(FingerprintStrategy):
   :API: public
   """
 
+  @deprecated('1.5.0.dev0',
+              'The information that was previously included in the fingerprint by '
+              'TaskIdentityFingerprintStrategy is now included by default in all Tasks.')
```

Comment:
Yep, agreed. Extending `FingerprintStrategy` should be sufficient... `Default` doesn't really do much.